easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb

```diff
@@ -0,0 +1,17 @@
+name = 'COMSOL'
+version = '5.2a'
+
+homepage = 'https://www.comsol.se/'
+description = """COMSOL Multiphysics - Simulation Tool for Electrical, Mechanical, Fluid Flow, and Chemical
+Applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+# License is required (EB_COMSOL_LICENSE), e.g:
+# EB_COMSOL_LICENSE="FFFFFFFFF-BBB-123456-123456"
+# EB_COMSOL_LICENSE="1718@licsvr1 1718@licsvr2"
+# EB_COMSOL_LICENSE="/path/to/license.dat"
```

Comment:
This should be updated according to the changes in https://github.com/easybuilders/easybuild-easyblocks/pull/1317?